yapf/yapflib/style.py

```diff
@@ -54,6 +54,13 @@ def SetGlobalStyle(style):
 _STYLE_HELP = dict(
     ALIGN_CLOSING_BRACKET_WITH_VISUAL_INDENT=textwrap.dedent("""\
       Align closing bracket with visual indentation."""),
+    ALIGN_ASSIGNMENT=textwrap.dedent("""\
+      Align assignment or augmented assignment operators.
+      If there is a blank line or newline comment or objects with newline entries in between,
+      it will start new block alignment."""),
+    NEW_ALIGNMENT_AFTER_COMMENTLINE=textwrap.dedent("""\
+      Start new assignment or colon alignment when there is a newline comment in between."""
+                                                   ),
```

Comment:
Please don't leave the ending paren in that place. :-)

Reply:
I'm also not a fan of this knob's name. Maybe something like `ALIGN_ASSIGNMENT_AFTER_COMMENTS`.